cluster-provision/gocli/cmd/provision.go

```diff
@@ -241,7 +257,7 @@ func provisionCluster(cmd *cobra.Command, args []string) (retErr error) {
 
 	envVars := fmt.Sprintf("version=%s slim=%t", version, slim)
 	if strings.Contains(phases, "linux") {
-		err = performPhase(cli, nodeContainer(prefix, nodeName), "/scripts/provision.sh", envVars)
+		err := sshClient.Command("sudo " + envVars + " /bin/bash /scripts/provision.sh")
```

Comment:
Are these scripts copied at this stage?

Reply:
no they weren't. i moved the copying to happen earlier